kubernetes/manifest-templates/velum.yaml

```diff
@@ -82,3 +90,6 @@ spec:
     - name: velum-source-code
       hostPath:
         path: ${project_dir}
+    - name: velum-secrets
+      hostPath:
+        path: /tmp/velum-secrets
```

Comment:
as discussed on IRC, the rails app is only mounted persistently in the dev environment, not in production. so we still need to find a folder.